src/styles/utils/variables.scss

```diff
@@ -0,0 +1,8 @@
+$color-primary: #00acdc;
+$linkBlueColor: #00acdc; // - без цієї змінної летить код. Не знайшов його використання ніде.
```

Comment:
delete it, and here
https://github.com/mate-academy/layout_catalog/pull/3551/files#diff-950e098ca83812275252fd2902a542325716e3e222214cd5714d860838f030b4R14